pydantic/plugin/loader.py

```diff
@@ -0,0 +1,54 @@
+"""This module is used to load plugins for pydantic.
+
+Plugins are installed via Python entry points, for example in `pyproject.toml`:
+
+    [tool.poetry.plugins."pydantic"]
+    observer = "pydantic_sdk:main"
```

Comment:
can you show the PEP 0621 way of doing it, e.g. [here](https://github.com/samuelcolvin/pytest-pretty/blob/af6cd38ea37e5a3879428f401804d1da72177c81/pyproject.toml#L55-L56), not poetry.

Or maybe both?

Reply:
this is the code you would use if you were using poetry, I'd prefer the main example to be how to do it if you're using a PEP 621 pyproject.toml.

E.g:

```toml
[project.entry-points.pydantic]
pydantic_sdk = "pydantic_sdk:main"
```

or whatever it is.

Reply:
Oh! I didn't even notice I was using poetry there... 😅